api/controllers/Depmu_EntryController.js

```diff
@@ -39,7 +39,7 @@ module.exports = {
         _.merge(prebooking, result)),
 
   filterContingencyWithoutCid: prebooking =>
-  !prebooking.contingency || (prebooking.contingency && _.isNull(prebooking.cid_id)),
+  !prebooking.contingency || prebooking.contingency && _.isNull(prebooking.cid_id),
 
```

Comment:
more for readability. Hopefully we are able to review our eslint setting at a later point, potential to update a few rules, particularly http://eslint.org/docs/rules/no-extra-parens